scripts/unpublished-changes.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+environment="${1:-production}"
+publishedrevision="$(npm run xpr:status \"${environment}\" | grep 'Job Id' | cut -f2 -d ':')"
```

Comment:
This assumes the user has added xpr:status to their package.json and that we're running through npm (which is not required). We should run exp-containerdeploy status directly from here instead.